app/src/androidTest/java/com/example/concordia_campus_guide/InstrumentalTests/LocationFragmentInstrumentalTest.java

```diff
@@ -51,9 +56,10 @@ public void init() {
 
     @Test
     public void createMarkerIcon(){
-        Bitmap bitmap1 = viewModel.createBitmapMarkerIcon("EV");
-        Bitmap bitmap2 = viewModel.createBitmapMarkerIcon("EV");
-        Bitmap bitmap3 = viewModel.createBitmapMarkerIcon("H");
+        DrawPolygons drawPolygons = new DrawPolygons();
```

Comment:
The class should be called polygonDrawer or something since verbs are used for methods normally

Reply:
done